nautobot_device_lifecycle_mgmt/templates/nautobot_device_lifecycle_mgmt/inc/device_notice.html

```diff
@@ -26,6 +26,13 @@
 
             </li>
             {% endif %}
+            {% if replacements %}
```

Comment:
This doesn't show up due to the below conditional at the top of the template:


```
{% if perms.nautobot_device_lifecycle_mgmt.view_hardwarelcm and 
hw_notices %}
```

In any case, I think we should move it to its own inc template.